src/LightDFG/services/LightDFG.jl

```diff
@@ -336,7 +336,7 @@ function savedot_attributes(io::IO, dfg::LightDFG)
         write(io, "$vl [color=red, shape=ellipse];\n")
     end
     for fl in listFactors(dfg)
-        write(io, "$fl [color=blue, shape=box];\n")
+        write(io, "$fl [color=blue, shape=box, fontsize=8, fixedsize=false, height=0.1, width=0.1];\n")
```

Comment:
Maybe the cause of this: #728

Reply:
That could make sense i did change this recently.  i wanted to shrink the factor boxes with GraphViz so added a few attributes.  Could we perhaps modify the test a little to allow that?

Reply:
My plan is to make time for coding on Friday, I'll update the tests if you haven't gotten to it by then.

Reply:
Linked issue and thanks for the fix!

Reply:
also see #735